defaults/main.yml

```diff
@@ -27,8 +27,10 @@ pretalx_mail_password: password
 pretalx_mail_tls: "False"
 pretalx_mail_ssl: "True"
 
-pretalx_nginx: false
-pretalx_nginx_path: false
+pretalx_nginx_conf_dir: false
```

Comment:
Could you please reverse the renaming of existing configuration variables? This would break any existing use of this role for users, and since ansible galaxy does not have any proper dependency management, I'd rather not introduce the risk.

Reply:
Good point, will do!